app/presenters/produce_price_presenter.rb

```diff
@@ -3,11 +3,12 @@ class ProducePricePresenter
   DATE = 'date'
   LABEL = 'label'
   PRICE = 'price'
+  UNIT = 'unit'
 
   def initialize(input, items: nil)
     @produce_prices = input
+    @items = items
     @response = {}
-    @response[:items] = items
```

Comment:
sorry i might have removed this in my example. we want to return the items so lets keep this